pages/_app.tsx

```diff
@@ -2,6 +2,7 @@ import "@/styles/globals.css";
 import type { AppProps } from "next/app";
 import { Providers } from "@/GlobalRedux/provider";
 import "material-icons/iconfont/material-icons.css";
+import "material-symbols";
```

Comment:
Is this the right way to use `material-symbols`?

Reply:
I refered the package documentation Sir.
![image](https://github.com/PiyushKalyanpy/GyanaGuru/assets/91585213/5df4b9ee-d617-44cc-9702-b46810780acd)